.github/workflows/axe-accessibility-check.yml

```diff
@@ -0,0 +1,30 @@
+name: axe
+on:
+  push:
+    branches: 
+      - master
+      - develop
+  pull_request:
+    branches: 
+      - master
+      - develop
+jobs:
+  axe:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [16.x, 18.x]
+        
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+      - run: npm ci
+      - run: npm run build --if-present
+      - run: npm start & npx wait-on http://localhost:4010
```

Comment:
We don't have a start script! https://github.com/camicroscope/caMicroscope/blob/dae72cfaaaccf40b82f1f75f16a2a80356a46722/package.json#L6

Reply:
It appears so. To add a start script, I need to know the entry point to the application. From my research, it is typically specified in the main field of the package.json file as "index.js" or "app.js" or "server.js". But I don't see it. I've scoured the entire repo and I can't find anything remotely close to it.

I was hoping you could point me in the right direction, please.